Tests/SonarScanner.MSBuild.TFS.Test/Resources/Expected.xmlcoverage

```diff
@@ -17,7 +17,7 @@
         </function>
       </functions>
       <source_files>
-        <source_file id="0" path="E:\ConsoleApp2\ConsoleApp2\Program.cs" checksum_type="unknown">
+        <source_file id="0" path="E:\ConsoleApp2\ConsoleApp2\Program.cs" checksum_type="SHA256" checksum="A4024AE875C549AB82B436EAA5F627860243A62D3F829EBE3052BCEEE7ACCDA8">
```

Comment:
The only difference between the two reports, except layout. See the diff linked in https://github.com/SonarSource/sonar-scanner-msbuild/pull/1737#issuecomment-1787526017 for details.